test/plugin_controller_test.dart

```diff
@@ -20,28 +20,28 @@ void main() {
     _MethodChannelMock _methodChannel;
     _ArgsToProtobufConverterMock _argsConverter;
     ProtobufConverter _protobufConverter;
-    _EventChannelMock _connectedDeviceChannel;
-    _EventChannelMock _argsChannel;
-    _EventChannelMock _scanChannel;
-    _EventChannelMock _statusChannel;
+    StreamController<List<int>> _connectedDeviceStreamController;
+    StreamController<List<int>> _argsStreamController;
+    StreamController<List<int>> _scanStreamController;
+    StreamController<List<int>> _statusStreamController;
 
     setUp(() {
       _argsConverter = _ArgsToProtobufConverterMock();
       _methodChannel = _MethodChannelMock();
       _protobufConverter = _ProtobufConverterMock();
-      _connectedDeviceChannel = _EventChannelMock();
-      _argsChannel = _EventChannelMock();
-      _scanChannel = _EventChannelMock();
-      _statusChannel = _EventChannelMock();
+      _connectedDeviceStreamController = StreamController();
+      _argsStreamController = StreamController();
+      _scanStreamController = StreamController();
+      _statusStreamController = StreamController();
```

Comment:
To my taste it's nice that we don't have to mess with (even mock) event channels in the tests.